plugins/base/src/main/java/edu/wpi/first/shuffleboard/plugin/base/data/fms/FmsInfo.java

```diff
@@ -69,7 +69,7 @@ public FmsInfo(Map<String, Object> map) {
         Maps.<String, Number>get(map, MATCH_NUMBER).intValue(),
         Maps.<String, Number>get(map, REPLAY_NUMBER).intValue(),
         MatchType.fromOrdinal(Maps.<String, Number>get(map, MATCH_TYPE).intValue()),
-        Maps.get(map, IS_RED_ALLIANCE) ? Alliance.RED : Alliance.BLUE,
+        (Boolean)Maps.get(map, IS_RED_ALLIANCE) ? Alliance.RED : Alliance.BLUE,
```

Comment:
This change seems irrelevant to this PR

Reply:
It failed to compile with if I didn't do this. Not sure why.

Reply:
Ok, must have just been a weird bug. Reverted.